### PR TITLE
OSDOCS-8494: Scalability and Performance Technology Preview

### PR DESCRIPTION
**Pull request by GroceryBoyJr**
Version(s): 4.15

Issue: https://issues.redhat.com/browse/OSDOCS-8494

Link to docs preview: https://71326--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes

QE review:
- [x] QE has approved this change.

Additional information:

image

### CODE REVIEW

**Comment by GroceryBoyJr** [comment]
@xltian Your QE review requested please.

**Comment by GroceryBoyJr** [comment]
@mcurry-rh your SME review requested please.

File: release_notes/ocp-4-15-release-notes.adoc [hunk 1]
```diff
@@ -1275,25 +1280,40 @@ In the following tables, features are marked with the following statuses:
...
 
 |Intel E810 Westport Channel NIC as PTP grandmaster clock
 |Technology Preview
 |Technology Preview
-|Technology Preview
+|General Availability
 
 |Workload partitioning for three-node clusters and standard clusters
 |Technology Preview
 |Technology Preview
```
Comment (qiliRedHat) [inline release_notes/ocp-4-15-release-notes.adoc:1315]: In https://docs.openshift.com/container-platform/4.14/release_notes/ocp-4-14-release-notes.html#ocp-4-14-technology-preview, 'Workload partitioning for three-node clusters and standard clusters' is already shown as 'General Availability'.

**Review by qiliRedHat — CHANGES_REQUESTED** [review]

File: release_notes/ocp-4-15-release-notes.adoc [hunk 2]
```diff
@@ -1275,25 +1280,40 @@ In the following tables, features are marked with the following statuses:
...
 
+|{sno-caps} cluster expansion with worker nodes
```
Comment (qiliRedHat) [inline release_notes/ocp-4-15-release-notes.adoc:1283]: For this and the following 2 features, I have a question. If all 3 releases are 'General Availability', what is the purpose to list them under 'Technology Preview features' section?

Comment (GroceryBoyJr) [reply release_notes/ocp-4-15-release-notes.adoc:1283]: @qiliRedHat  Thank you for asking! We still support the last four releases and we have customers on those releases, therefore we must show the history of the feature in past and current release notes.  All sections have the name "Technology Preview features" and list the features even after they have gone General Availability.  We do this for legal requirements and for tech support requirements.

File: release_notes/ocp-4-15-release-notes.adoc (hunk 1 above)
Comment (GroceryBoyJr) [reply release_notes/ocp-4-15-release-notes.adoc:1315]: Thank you @qiliRedHat ! I will correct this!

File: release_notes/ocp-4-15-release-notes.adoc [hunk 3]
```diff
@@ -1275,25 +1280,40 @@ In the following tables, features are marked with the following statuses:
...
+|NUMA-aware scheduling with NUMA Resources Operator
+|General Availability
+|General Availability
+|General Availability
+
 |Mount namespace encapsulation
 |Technology Preview
 |Technology Preview
-|Technology Preview
+|General Availability
```
Comment (rsevilla87) [inline release_notes/ocp-4-15-release-notes.adoc:1301]: According to the docs this feature is still TP 
https://github.com/openshift/openshift-docs/blob/enterprise-4.15/modules/enabling-encapsulation.adoc#configuring-mount-namespace-encapsulation

Comment (GroceryBoyJr) [reply release_notes/ocp-4-15-release-notes.adoc:1301]: Thank you @rsevilla87 !

**Comment by GroceryBoyJr** [comment]
I have removed the Intel E810 PTP Grandmaster clock section from this chart because it already occurs in the Networking chart. Requested corrections from SME and QE have been completed.

**Comment by GroceryBoyJr** [comment]
I have updated the table to put the items in alphabetic order. @rsevilla87 and @qiliRedHat I seek your review please.

**Comment by qiliRedHat** [comment]
@GroceryBoyJr I've reviewed again, my comment is addressed, thanks.

**Comment by xltian** [comment]
ACK

**Comment by sabrinajess** [comment]
LGTM

**Pull request merged**